BraintreeCore/src/main/java/com/braintreepayments/api/Authorization.java

```diff
@@ -59,15 +59,15 @@ public String toString() {
         return rawValue;
     }
 
-    private static boolean isTokenizationKey(String tokenizationKey) {
```

Comment:
Drop-in uses these methods so making them package-private

Reply:
My understand of package-private is that it can only be seen and used by the package in which it was declared. So doesn't this mean drop-in can't use it then, and it would need to be public? (Or protected if something in Drop-In subclasses `Authorization`)?

Reply:
We refactoring Drop-in to follow the same package structure as core so that we can use the package-private methods there as well.

Reply:
Ah - I didn't remember that. Thanks!